auto-rollback/http/http.go

```diff
@@ -3,81 +3,73 @@ package http
 import (
 	"encoding/json"
 	"errors"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"net/netip"
-	"strings"
 )
 
-func FetchLatestOrdinal(blockExplorerUrl string) (uint64, error) {
-	var url string
-	if strings.HasSuffix(blockExplorerUrl, "/") {
-		url = blockExplorerUrl
+func ToUrl(ip netip.AddrPort) string {
+	return "http://" + ip.String() + "/"
+}
+
+func FetchNodeHealth(ip netip.AddrPort) error {
+	url := ToUrl(ip) + "node/health"
+	resp, err := http.Get(url)
+
+	if err != nil {
+		return err
```

Comment:
I think it also means that the node is not healthy 🤔

Reply:
Yes. That's why I return err then :P